pkg/console/starter/starter.go

```diff
@@ -259,6 +242,9 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 			"Progressing",
 			"Available",
 			// "Faililng"
+			// added in 4.3, can remove in 4.4, to ensure operators running 4.3.x are not wedged.
```

Comment:
I'm not sure if this change is strictly necessary.  These were new conditions with the router-ca sync commit within the 4.3 time frame, so if the code is not running in production we can ignore.

Reply:
yeah not sure we need this change, if not would rather not add it...

Reply:
I would take this out, and we can backport the revert to 4.3.